packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -44,32 +44,32 @@ class ServiceConnectionManager {
     _serviceExtensionManager = ServiceExtensionManager(isolateManager);
   }
 
-  final StreamController<VmServiceWrapper> _connectionAvailableController =
-      StreamController<VmServiceWrapper>.broadcast();
+  final StreamController<VmServiceWrapper?> _connectionAvailableController =
+      StreamController<VmServiceWrapper?>.broadcast();
 
   Completer<VmService> _serviceAvailable = Completer();
 
   Future<VmService> get onServiceAvailable => _serviceAvailable.future;
 
   bool get isServiceAvailable => _serviceAvailable.isCompleted;
 
-  VmServiceCapabilities _serviceCapabilities;
-  VmServiceTrafficLogger serviceTrafficLogger;
+  VmServiceCapabilities? _serviceCapabilities;
+  VmServiceTrafficLogger? serviceTrafficLogger;
 
-  Future<VmServiceCapabilities> get serviceCapabilities async {
+  Future<VmServiceCapabilities?> get serviceCapabilities async {
```

Comment:
this should return non-nullable VmServiceCapabilities

Reply:
done